tdvt/tdvt/config_gen/gentests.py

```diff
@@ -37,12 +37,12 @@ def get_customized_table_name(attributes, base_table):
 
     t = Template(table_name)
     #These substitution holders are in templates.py.
-    if 'tablenameUpper' in attributes:
+    if 'tablenameUpper' in attributes and attributes['tablenameUpper'] == 'True':
```

Comment:
Backwards compatibility is probably the biggest consideration here.  Previously any value was considered true.  As written this is the most correct implementation though.

The fewest breaking changes that we only consider 'key' == 'False' or 'key' not specified to be the false cases.  I'm interested in others thoughts as well here.

Either way its worth pulling this out as a helper function so we can make the change once and possibly adjust it over time based on feedback too.

Reply:
Changed this to a helper function.
Backwards compatibility is a good point, looking at [all tdvt ini files ](https://opengrok.prod.tableautools.com/source/search?project=teams_near&full=LogicalConfig&defs=&refs=&path=&type=&xrd=&nn=1&si=full&si=full)have logical config in monolith, they all use the correct format, so they should be good.
so the breaking cases are empty value or any values except for True, I think it is hard to make this change and also backwards compatible since the goal is basically to make the boolean value here works as expected.